client/src/components/main/CreateNewPost.jsx

```diff
@@ -46,6 +46,7 @@ const CreateNewPost = (props) => {
     // listen for location selection from the dropdown
     google.maps.event.addListener(autocomplete, 'place_changed', () => {
       place = autocomplete.getPlace();
+      console.log(place);
```

Comment:
Is this console.log necessary in our master branch??

Reply:
No it's not. But we do have a bunch of console logs in master currently

Reply:
Do you mind taking it out?? It's just one more piece of code cleanup that we won't have to chase down later. Thanks.

Reply:
Console logging data i think is unnecessary but console logging error/success status as actions execute to show the current state is ok.